examples/nlp/token_classification/punctuation_capitalization.py

```diff
@@ -164,11 +190,11 @@ def create_pipeline(
     capit_label_ids=None,
     ignore_extra_tokens=args.ignore_extra_tokens,
     ignore_start_end=args.ignore_start_end,
-    use_cache=args.use_cache,
+    overwrite_processed_files=args.overwrite_processed_files,
```

Comment:
Would you please print something when overwrite_processed_files is false and the processed files are already exist, to let the user know that you have not created the files?

Reply:
Such info message is logged from the dataset